src/app/operators/dialogs/csv/csv-config/csv-properties/csv-properties.component.ts

```diff
@@ -205,9 +205,19 @@ export class CsvPropertiesComponent implements OnInit, AfterViewInit, OnDestroy
             }),
             this.propertiesService.header$.subscribe(h => {
                 this.header = h;
+
+                // Refactor user options(disable options that are not possible anymore).
```

Comment:
Extract a function instead of writing a comment.
`disableInvalidOptions();` or something like this